mailpoet/assets/js/src/automation/integrations/mailpoet/analytics/components/tabs/emails/rows.tsx

```diff
@@ -82,56 +79,49 @@ export function transformEmailsToRows(emails: EmailStats[]) {
                 <a
                   href={`?page=mailpoet-newsletters#/sending-status/${email.id}`}
                 >
-                  {`${email.sent.current}`}
+                  {`${email.sent}`}
                 </a>
               </Tooltip>
             }
             subValue={
               // Shows the percentage of sent emails compared to the previous email
               percentageFormatter.format(
-                calculatePercentage(
-                  email.sent.current,
-                  email.sent.previous,
-                  true,
-                ) / 100,
+                calculatePercentage(email.sent, email.sent, true) / 100,
```

Comment:
Since both values passed to the function are the same, it will always return 100%. Is it intended? Is it meaningful?

Reply:
Great catch! Thank you. f74056e38 was a bit hasty. In case of `sent` we want to actually compare with the amount of emails sent in the previous period. Addressed in 0d094ecbe